spec/fixtures/expect/dir/legacy.js

```diff
@@ -0,0 +1,6 @@
+import legacy from "../../../actual/dir/legacy.js";
+import * as legacy2 from "../../../actual/dir/legacy.js";
```

Comment:
So it should not be: `../../../actual/`

Reply:
not following, why are you saying it should not be "../../../actual/" ?

Reply:
It seems that there should be `./legacy.js` and not `../../../actual/dir/legacy.js`. The directory `actual` is an artifact.

Reply:
"actual" isn't an artifact, in this case (and my use case), actual = src, dist = lib... both the src and lib directory would be in the production deploy, where lib was transpiled TS files, and JS is original node compatible JS. the test is outputing as i would expect, just looks weird because lib & src are not in the project root, but under a spec directory